Sources/Actions/UnreadReplies.php

```diff
@@ -54,44 +54,6 @@ class UnreadReplies extends Unread
 	 */
 	protected bool $is_topics = false;
 
-	/****************************
-	 * Internal static properties
-	 ****************************/
-
-	/**
-	 * @var self
-	 *
-	 * An instance of this class.
-	 * This is used by the load() method to prevent multiple instantiations.
-	 */
-	protected static UnreadReplies|Unread $obj;
-
```

Comment:
Just tested this again to triple-check that it still works. Both variations of the unread topics actions load because in the trait we use late static binding in the `call()` method.

> The important difference from the user's point of view is that given these three lines of code:
> 
> ```php
> $someObject->methodOne();
> self::methodTwo();
> static::methodThree();
> ```
> 
> * The definition of methodOne used will be whichever class $someObject is an instance of at the time that code runs. It might be a different method each time the line of code runs, if $someObject has a different value. This is **late** binding.
> * It will also reference the _specific instance_ in $someObject, placing it in the magic variable $this. It is a **non-static** call.
> * The definition of methodTwo used will be the one in *the class where that line of code is written*. Every time that line of code runs, it will reference the same class. This is **early** binding.
> * The definition of methodThree used will depend on how the code was called - if it was called with the name of a child class with its own version of methodThree, that version will be used. Like methodOne, the line might run a different method each time. This is **late** binding.
> * Both methodTwo and methodThree will only reference a _class_, not an _instance_. They will not populate the magic variable $this. They are **static** calls.

https://stackoverflow.com/a/67270186/4710434

Reply:
You are right. Fair enough then. 🙂